modules/mono/editor/bindings_generator.cpp

```diff
@@ -953,7 +953,9 @@ void BindingsGenerator::_generate_global_constants(StringBuilder &p_output) {
 	p_output.append("namespace " BINDINGS_NAMESPACE ";\n\n");
 
 	p_output.append("\n#pragma warning disable CS1591 // Disable warning: "
-					"'Missing XML comment for publicly visible type or member'\n");
+					"'Missing XML comment for publicly visible type or member'\n"
+					"#pragma warning disable CA1716 // Disable warning: "
+					"'Identifiers should not match keywords'\n");
```

Comment:
I don't know how to feel about this. I would like to keep this in sight. We may want to rename those in the future. The Object one if fine to disable, because it's very obvious.

Reply:
Can this be done inside an `#if`? If so, we could have a configuration to enable this warning, while having it disabled by default.

Reply:
It can and it sounds good to me. I don't really want to suppress these warnings but solving them properly would require renaming which seems unlikely to happen in a timely manner and after the beta we won't want to break compatibility.